lib/eztemplate/classes/eztemplatetypeoperator.php

```diff
@@ -276,7 +276,7 @@ function isTransform( $operatorName, &$node, $tpl, &$resourceData,
 
             case $this->GetClassName:
             {
-                $code .= 'strtolower( get_class( %1% ) );';
+                $code .= 'strtolower( is_object( %1% ) ? get_class( %1% ) : "" );';
```

Comment:
```suggestion
                $code .= 'is_object( %1% ) ? strtolower( get_class( %1% ) ) : "";';
```

Reply:
Wow, you are fast!
I understand what you mean: In case of an error get_class returns false.
But in this case we need an value which is passed to strtolower function, so I thought it would be more appropriate to use an empty string.

Reply:
Sorry @blankse for the confusion, you're right. Forget my last comment ; )